docs/topics/multiplatform/multiplatform-full-stack-app.md

```diff
@@ -0,0 +1,991 @@
+    ConnectionString("$it?retryWrites=false")
+}
+
+val client =
+    if (connectionString != null) KMongo.createClient(connectionString).coroutine else KMongo.createClient().coroutine
+val database = client.getDatabase(connectionString?.database ?: "shoppingList")
+```
+
+This ensures that the `client` is created based on this information whenever the environment variables are set.
+Otherwise (for instance, on `localhost`), the database connection is instantiated as previously.
+
+### Create the Procfile
+
+Managed cloud platforms like Heroku or PaaS implementations like [Dokku](https://github.com/dokku/dokku) also handle
+the lifecycle of your application. To do so, they require an "entry point" definition. These two platforms use
+a file called `Procfile` that you have in the project root directory. It points to the output generated by the `stage` task
+(which is included in the Gradle template already):
+
+```shell
+web: ./build/install/shoppingList/bin/shoppingList
+```
+
+### Turn on production mode
+
+To turn on a compilation with optimizations for the JavaScript assets, pass another flag to the build
+process. In the **Run/Debug Configurations** menu, set the environment variable `ORG_GRADLE_PROJECT_isProduction` to `true`.
+You can set this environment variable when you deploy the application to the target environment.
+
+> You can find the finished application on GitHub on the [`final` branch](https://github.com/kotlin-hands-on/jvm-js-fullstack/tree/final).
+>
+{type="note"}
+
+#### Relevant Gradle configuration {initial-collapse-state="collapsed"}
+
+The `stage` task is an alias for `installDist`:
+
+```kotlin
+// Alias "installDist" as "stage" (for cloud providers)
+tasks.create("stage") {
+    dependsOn(tasks.getByName("installDist"))
+}
+
+// only necessary until https://youtrack.jetbrains.com/issue/KT-37964 is resolved
+distributions {
+    main {
+        contents {
+            from("$buildDir/libs") {
+                rename("${rootProject.name}-jvm", rootProject.name)
+                into("lib")
+            }
+        }
+    }
+}
+```
+
+## What's next
+
+#### Add more features {initial-collapse-state="collapsed"}
+
+See how your application could still be expanded and improved:
+
+* **Improve the design**. You could make use of `styled-components`, one of the libraries that have Kotlin wrappers
+  provided. If you want to see `styled-components` in action, look at
+  the [Build a web application with React and Kotlin/JS](js-react.md) tutorial.
+* **Add crossing out list items**. For now, list items just vanish with no record of them existing. Instead of deleting
+  an element, use ~~crossing out~~.
+* **Implement editing**. So far, an entry in the shopping list can't be edited. Consider adding an edit button.
+
+#### Join the community and get help {initial-collapse-state="collapsed"}
+
+You can join the official [Kotlin Slack](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up). There, you can
```

Comment:
We could change this to https://slack-chats.kotlinlang.org/ and link the respective channels from there